20231804T160439/index.md

```diff
@@ -0,0 +1,394 @@
+---
+title: 'Relacionamento das bases no relatório operacional: método Concatenate x Linktable'
+---
+
+# Introdução 
```

Comment:
Algumas referências interessantes que encontrei pra ajudar nessa contextualização inicial:

- [The Soul of the Data Warehouse, Part 2: Drilling Across - Kimball Group](https://www.kimballgroup.com/2003/04/the-soul-of-the-data-warehouse-part-two-drilling-across/)
- [Consolidated Fact Table | Kimball Dimensional Modeling Techniques](https://www.kimballgroup.com/data-warehouse-business-intelligence-resources/kimball-techniques/dimensional-modeling-techniques/consolidated-fact-table/)
- [No Detail Too Small - DecisionWorks Consulting](https://decisionworks.com/2003/10/no-detail-too-small/)